tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/default/uuid/field_uuid_default_model_test.dart

```diff
@@ -0,0 +1,267 @@
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+        class: Example
+        table: example
+        fields:
+          uuidInvalid: UuidValue?, defaultModel=INVALID_UUID
+        ''',
+          ).build()
+        ];
+
+        var collector = CodeGenerationCollector();
+        StatefulAnalyzer(config, models, onErrorsCollector(collector))
+            .validateAll();
+
+        expect(collector.errors, isNotEmpty);
+
+        var firstError = collector.errors.first as SourceSpanSeverityException;
+        expect(
+          firstError.message,
+          'The "defaultModel" value must be a valid UUID string or "random" (e.g., "defaultModel"=random or "defaultModel"="550e8400-e29b-41d4-a716-446655440000").',
+        );
+      },
+    );
+
+    test(
+      'when the field is of type UUID with a malformed UUID in single quotes, then an error is generated',
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+          class: Example
+          table: example
+          fields:
+            uuidMalformed: UuidValue?, defaultModel='550e8400-e29b-41d4-a716-INVALID'
+          ''',
+          ).build()
+        ];
+
+        var collector = CodeGenerationCollector();
+        StatefulAnalyzer(config, models, onErrorsCollector(collector))
+            .validateAll();
+
+        expect(collector.errors, isNotEmpty);
+
+        var firstError = collector.errors.first as SourceSpanSeverityException;
+        expect(
+          firstError.message,
+          'The "defaultModel" value must be a valid UUID (e.g., "550e8400-e29b-41d4-a716-446655440000").',
+        );
+      },
+    );
+
+    test(
+      'when the field is of type UUID with a malformed UUID in double quotes, then an error is generated',
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+          class: Example
+          table: example
+          fields:
+            uuidMalformed: UuidValue?, defaultModel="550e8400-e29b-41d4-a716-INVALID"
+          ''',
+          ).build()
+        ];
+
+        var collector = CodeGenerationCollector();
+        StatefulAnalyzer(config, models, onErrorsCollector(collector))
+            .validateAll();
+
+        expect(collector.errors, isNotEmpty);
+
+        var firstError = collector.errors.first as SourceSpanSeverityException;
+        expect(
+          firstError.message,
+          'The "defaultModel" value must be a valid UUID (e.g., "550e8400-e29b-41d4-a716-446655440000").',
+        );
+      },
+    );
+
+    test(
+      'when the field is of type UUID with non-UUID defaultModel value, then an error is generated',
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+          class: Example
+          table: example
+          fields:
+            uuidType: UuidValue?, defaultModel=test
```

Comment:
Question: Is there a difference between this test and the one where `INVALID_UUID` is specified?

Reply:
No, just adding extra test, should this or the other one be removed ?

Reply:
Done!